mypy/myunit/ick.py

```diff
@@ -0,0 +1,6 @@
+"""Ick, mutable globals!
```

Comment:
Style nit: Use the last `""""` on the same line as the first one for one-line docstrings, i.e. like this:

```
"""Ick, mutable globals!"""
```